pages/scripts/delayed.js

```diff
@@ -124,9 +97,25 @@ const otId = placeholders.onetrustId;
 if (otId) {
   const cookieScript = loadScript('https://cdn.cookielaw.org/scripttemplates/otSDKStub.js');
   cookieScript.setAttribute('data-domain-script', `${otId}${isProd ? '' : '-test'}`);
-  cookieScript.setAttribute('data-dlayer-name', 'adobeDataLayer');
+  cookieScript.setAttribute('data-dlayer-name', 'dataLayer');
```

Comment:
Is this needed for something if we no longer push to the dataLayer?

Reply:
Probably not, but I am not sure